templates/taxi/index.html

```diff
@@ -0,0 +1,19 @@
+{% extends "base.html" %}
+
+{% block sidebar %}
+    <h6>Sidebar</h6>
+    <ul>
+        <li><a href="{% url 'taxi:index' %}">Home</a></li>
+        <li><a href="#">All ars</a></li>
+        <li><a href="#">All drivers</a></li>
+        <li><a href="#">All manufacturers</a></li>
+    </ul>
+{% endblock %}
```

Comment:
sidebar should be placed in a different file in `includes` directory and included in` base.html`